Applications/PlazaDetails/Strength.qmd

```diff
@@ -1,27 +1,32 @@
 ---
-title: "Credit Strength"
+title: "Credit Consumption"
 aliases: 
   - Application/PlazaDetails/Strength.html
 ---
 
-The following mentioned are among the popular services available in openEO algorithm plaza. Here we have presented an average credit strength for these services. Please note that depending on the time interval and area of interest, the credits can slightly vary.
+The following are among the popular services available in OpenEO Algorithm Plaza. Here, we have presented an average credit strength for these services. Please note that the credits can vary slightly depending on the time interval and area of interest.
```

Comment:
I think you correctly reference to the spatio-temporal 'location' for a fixed size. It might however cause some confusion since the time interval and area of interest also sound like you can indeed change the size.

Therefore, I would simply remove 'slightly'.